Firestore/core/test/firebase/firestore/FSTGoogleTestTests.mm

```diff
@@ -186,6 +188,8 @@ @interface GoogleTests : XCTestCase
  * this way here allows XCTAssert and friends to work.
  */
 void ReportTestResult(XCTestCase* self, SEL _cmd) {
```

Comment:
Question: is this function called after the test was supposed to finish running? Or is using it to kick off the actual tests idiomatic?